igvm/cli.py

```diff
@@ -356,6 +357,26 @@ def parse_args():
         help='Shutdown VM, if running',
     )
 
+    subparser = subparsers.add_parser(
+        'evacuate',
+        description=evacuate.__doc__,
+    )
+    subparser.set_defaults(func=evacuate)
+    subparser.add_argument(
+        'hv_hostname',
+        help='Hostname of the hypervisor',
+    )
+    subparser.add_argument(
+        '--dry-run',
+        action='store_true',
+        help='Do not migrate but just print what would be done'
+    )
+    subparser.add_argument(
+        '--offline',
+        nargs='*',
+        help='Migrate VMs offline',
```

Comment:
This help message is not really that helpful. First I wondered why it is `nargs=*` but only then I noticed it is compared against server functions. Yet the help message suggests it is a boolean.